end-to-end-tests/specs/home.spec.js

```diff
@@ -315,12 +318,17 @@ describe('single study query', function() {
         it('should show mutations plot', function() {
             browser.url(`${CBIOPORTAL_URL}/index.do?cancer_study_id=ov_tcga_pub&Z_SCORE_THRESHOLD=2.0&RPPA_SCORE_THRESHOLD=2.0&data_priority=0&case_set_id=ov_tcga_pub_cna_seq&gene_list=BRCA1+BRCA2&geneset_list=+&tab_index=tab_visualize&Action=Submit&genetic_profile_ids_PROFILE_MUTATION_EXTENDED=ov_tcga_pub_mutations&genetic_profile_ids_PROFILE_COPY_NUMBER_ALTERATION=ov_tcga_pub_gistic`);
 
+            // TODO: wait for store to be populated (this is actual bug when
+            // someone clicks too fast on the next tab)
+            waitForOncoprint(10000);
+
             // click enrichments tab
             $('#enrichments-result-tab').waitForExist(30000);
+            browser.pause(2000); // wait for click handler
```

Comment:
im not sure if this is really true re click handler. It might have been something else that fixed the problem, but too much of a pain to find out